0008/README.md

```diff
@@ -0,0 +1,67 @@
+---
+id: 0008
+title: Explicit Certification For Components
+status: ideation
+authors: Jason DeTiberus <jdetiberus@packet.com>
+---
+
+## Summary
+
+Currently certificate configuration for Tinkerbell is handled through a
+combination of the `TINKERBELL_TLS_CERT` env variable and file based lookup.
+The file based lookup will use the `TINKERBELL_CERTS_DIR` env variable if
+defined, defaulting to `/certs/<facility>` if undefined. The certificate files
+themselves are hardcoded to `bundle.pem`, which is expected to contain both the
+CA certificate and the serving certificate, and `sserver-key.pem` for the
+private key.
+
+The current configuration methods do not offer a lot of flexibility for users,
+especially if they are looking to run Tinkerbell in Kubernetes and leverage
+automated certificate management tooling, such as
+[cert-manager](https://cert-manager.io/), for automated certificate generation
+and rotation.
+
+This proposal is intended to provide generalized guidance on how Tinkerbell
+components should expose certificate configuration to users.
+
+## Goals and not Goals
+
+Goals:
+
+1. Provide common methods of exposing certificate configuration for Tinkerbell
+services
+
+Non-Goals:
+
+1. Add tls support anywhere it is not currently enabled
+
+## Content
+
+Rather than using a combination of env variables and hard-coded defaults for
+the certificate file lookups, we can leverage explictly configured command line
+arguments for specifying the certificate path, and default certificate file
+names to use.
```

Comment:
there's no mention of env or files to set flags, neither here nor below. It should be mentioned right?